requirements.txt

```diff
@@ -6,7 +6,7 @@ numpy>=1.9
 matplotlib>=2.0,<3.8
 scipy>=1.10
 pandas>=1.3.4,<2.0
-astropy>=5.3.3
+astropy>=4.3.1
```

Comment:
I echo Sahil's question about the astropy version here.

Reply:
The reason is just related to the installation through docker. While I was testing I often got an error that pip could not found any compatible version. However changing the minimum version no error was raised again.